source/ed/data.cpp

```diff
@@ -218,6 +218,43 @@ void Data::shift_stop_times() {
     }
 }
 
+bool compare(const std::pair<idx_t, size_t>& p1, const std::pair<idx_t, size_t>& p2){
```

Comment:
`static bool compare` -> add `static` please (as it will not be used by other files).